workers/wfsfeature-harvester/src/main/java/org/fao/geonet/harvester/wfsfeatures/WFSHarvesterApi.java

```diff
@@ -85,6 +86,7 @@ public JSONObject indexWfs(
         produces = MediaType.ALL_VALUE,
         method = RequestMethod.DELETE)
     @ResponseStatus(value = HttpStatus.OK)
+    @PreAuthorize("hasAuthority('Editor')")
```

Comment:
Thanks, are any changes needed to the user interface? Or is the option to scan WFS only shown to editors?

Reply:
Cf. https://github.com/geonetwork/core-geonetwork/blob/main/web-ui/src/main/resources/catalog/components/viewer/wfsfilter/partials/wfsfilterfacet.html#L61-L62